.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+Please do **NOT** use the issue tracker for support questions. Any questions or comments of that nature can be answered on our support server located [here](https://discord.gg/red).
```

Comment:
I strongly disagree with this. A issue tracker can and should also be for support. Its just that nobody wants to do them. However! This is actually one of the best tools we have available to us. People will mostly Google first before going to a chat.

Reply:
I agree, I think we should allow both methods.
The advantage of directing them to our chat as well is that there is a possibility they may become enthusiastic new members of our community, which lives in the Discord anyway.

Reply:
I agree with Palm. Support questions should either be answered in the docs or asked in the support server. I am fully behind the idea that GH issues should be for code bugs/feature suggestions only. @Twentysix26 your time as tie-breaker has come lol

Reply:
For reference, I didn't pull that bit about not using issues for support questions out of nowhere. It's literally in [here](https://github.com/Cog-Creators/Red-DiscordBot/blob/V3/develop/.github/CONTRIBUTING.md#what-kinds-of-contributions-are-we-looking-for) which has been in for 3 months now

Reply:
I think support questions are fine as issues too, that's how it works in most projects. And I agree with @Kowlin , they have the advantage that solutions to common issues *will* show up on Google too. Not to mention that they can be referenced in Discord to avoid repeating the same steps everytime.  
I don't see a reason to limit issues to code topics only, IMO it would do more harm than good.